app/screens/about/about.js

```diff
@@ -207,6 +207,14 @@ export default class About extends PureComponent {
                         {licensee}
                         {learnMore}
                         <FormattedText
+                            id='mobile.about.powered_by'
```

Comment:
this should be shown only when the app is not built by Mattermost, and the localization string is missing in `en.json`